### PR TITLE
[topgen] Inter-module signal format

### PR DESCRIPTION
**Pull request by eunchan**
Revised the signal format to indicate the requester -> responder.

This is related to #42

### CODE REVIEW

**Comment by eunchan** [comment]
Hum.. I am trying to figure out why generated hjson files have so many changes. the Dict order may be different based on the python version. Am running this inside the docker.

**Comment by eunchan** [comment]
> Hum.. I am trying to figure out why generated hjson files have so many changes. the Dict order may be different based on the python version. Am running this inside the docker.

OK . The symptom is, if I run the topgen inside the docker, it changes the dict order randomly. In newer version at my workstation, the order is deterministic. @imphil Is this also same in Azure?

**Comment by imphil** [comment]
The dict ordering is undefined up to Python 3.7, `OrderedDict` instead if you want to keep the diffs in a specific ordering.

**Comment by eunchan** [comment]
Thanks Philipp, I will try to revise to use OrderedDict.

File: util/topgen/intermodule.py [hunk 1]
```diff
@@ -8,6 +8,36 @@
 from .lib import *
 
 
+def intersignal_format(uid: int, req: Dict, rsp: Dict) -> str:
+    """Determine the signal format of the inter-module
```
Comment (msfschaffner) [inline util/topgen/intermodule.py]: Maybe add ...inter-module **connection**?

**Comment by tjaychen** [comment]
thanks for this! i'm sure not everyone will agree that this reads as cleaner, but it reads cleaner to me :)

**Pull request merged**